youtube_dl/extractor/lrt.py

```diff
@@ -17,78 +15,66 @@ class LRTIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?lrt\.lt/mediateka/irasas/(?P<id>[0-9]+)'
     _TESTS = [{
         # m3u8 download
-        'url': 'http://www.lrt.lt/mediateka/irasas/54391/',
-        'md5': 'fe44cf7e4ab3198055f2c598fc175cb0',
+        'url': 'https://www.lrt.lt/mediateka/irasas/2000078895/loterija-keno-loto',
+        # md5 for first 10240 bytes of content
+        'md5': '8e6f0121ccacc17d91f98837c66853f0',
         'info_dict': {
-            'id': '54391',
```

Comment:
Do not touch existing tests.

Reply:
Hi Sergey! Can you elaborate what you mean here? Data structure has completely changed in lrt.lt service JSON responses, so I cannot keep existing tests because they will not be testing anything relevant.